configure.ac

```diff
@@ -127,20 +141,13 @@ case "$target" in
         )
         ;;
 esac
+fi
 
 AC_SUBST(MATHLIB)
 LIBS="$MATHLIB $LIBS"
 AC_SUBST(GL_LIBS)
 LIBS="$GL_LIBS $LIBS"
 
-dnl *******************************************
```

Comment:
this is harmless to keep no?

Reply:
With the proposed change, LIBS="$GL_LIBS $LIBS" is required.